webapp/packages/plugin-connections/src/ConnectionAuthService.ts

```diff
@@ -104,7 +104,7 @@ export class ConnectionAuthService extends Dependency {
       if (resetCredentials && isConnectedInitially && result === DialogueStateResult.Rejected) {
         await this.connectionInfoResource.init(key);
       }
-    } else {
+    } else if (!connection.connected) {
       await this.connectionInfoResource.init({ projectId: key.projectId, connectionId: key.connectionId });
     }
```

Comment:
this is not true fix i think, because this change is hides case but not solves the problem.
I suppose that problem is that one of extra requests to inner database resources silently re-opens connection on the backend. We need to investigate this case and solve the problem properly